src/components/ErrorMessage.test.tsx

```diff
@@ -1,10 +1,10 @@
 import React from "react";
 import renderer from "react-test-renderer";
 
-import Page from "./Page";
+import ErrorMessage from "./ErrorMessage";
 
 it("renders correctly", () => {
-  const component = renderer.create(<Page />);
+  const component = renderer.create(<ErrorMessage />);
 
   expect(component).toMatchSnapshot();
 });
```

Comment:
Let's also add a snapshot test with all the props provided. I usually go for `renders correctly without optional props` and `renders correctly with all props`.